nuxeo-drive-client/nxdrive/osi/windows/windows.py

```diff
@@ -5,18 +5,21 @@
 from nxdrive.logging_config import get_logger
 import os
 import sys
+import platform
 if sys.platform == "win32":
     import _winreg
 log = get_logger(__name__)
 
 
 class WindowsIntegration(AbstractOSIntegration):
     RUN_KEY = 'Software\\Microsoft\\Windows\\CurrentVersion\\Run'
+    EXPLORER = 'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Explorer'
 
     def __init__(self, manager):
         super(WindowsIntegration, self).__init__(manager)
         from nxdrive.osi.windows.win32_handlers import WindowsProcessFileHandlerSniffer
         self._file_sniffer = WindowsProcessFileHandlerSniffer()
+        self.guid = '{800B7778-1B71-11E2-9D65-A0FD6088709B}'
```

Comment:
I have updated the code to use engineId instead of productId, Using engineId helps to create separate shortcut for each account.